setup-jest.ts

```diff
@@ -0,0 +1,9 @@
+import { toMatchFile } from 'jest-file-snapshot';
+import { disconnect } from '#utils/network/backend/redis/redis-storage';
+import 'dotenv/config';
+
+expect.extend({ toMatchFile });
+
+afterAll(async () => {
+  await disconnect();
+});
```

Comment:
Otherwise, the handler is left hanging, and jest doesn't stop. It's not a problem locally, as you can SIGINT the process, but on the CI, the step never ends.

Reply:
Oh nice